crates/sovereign-sdk/full-node/sov-stf-runner/tests/runner_initialization_tests.rs

```diff
@@ -56,10 +56,21 @@ fn initialize_runner(
     MockProverService,
     sov_modules_api::default_context::DefaultContext,
 > {
+    let da_storage_path = storage_path.join("da").to_path_buf();
+    let rollup_storage_path = storage_path.join("rollup").to_path_buf();
+
+    if !std::path::Path::new(&da_storage_path).exists() {
+        std::fs::create_dir(da_storage_path.clone()).unwrap();
```

Comment:
No need to unwrap because there could be a dir created after you checked it for non-existence.